user_docs/en/userGuide.t2t

```diff
@@ -1411,6 +1411,15 @@ However, in order for you to read the context (i.e. that you are in a list and t
 
 To toggle focus context presentation from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+==== Interrupt speech while scrolling ====[BrailleSettingsInterruptSpeech]
+This setting determines if speech should be interrupted when the Braille display is scrolled backwards/forwards.
+Previous/next line commands are not affected.
+
+Speech may be found distracting while reading Braille.
+As such, this option is enabled by default, interrupting speech when scrolling.
+
+Disabling this option allows speech to be read along with Braille.
+

```

Comment:
A general question:
- **Is "interrupting" a clear enough term for user?**
- **Are there any/few/many prior usages of "Interrupt" is there with regards to speech?**
- - **Would "stopping/stopped/cancelled" be clearer?**

```suggestion
This setting determines if speech should be interrupted when the Braille display is scrolled backwards/forwards.
Previous/next line commands always interrupt speech.

On-going speech might be a distraction while reading Braille.
For this reason the option is enabled by default, interrupting (stopping) speech when scrolling braille.

Disabling this option allows speech to be heard while simultaneously reading Braille.
```

Reply:
For other example of the word "interrupt" with speech, see the following options:
* [Speech interrupt for typed characters](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#KeyboardSettingsSpeechInteruptForCharacters)
* [Speech interrupt for Enter key](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#KeyboardSettingsSpeechInteruptForEnter)